packages/jspsych/src/modules/metadata/metadata.puml

```diff
@@ -0,0 +1,79 @@
+@startuml Metadata
```

Comment:
I don't think we want the puml file to be tracked. Could add to gitignore?

Reply:
Added.